Banking/src/main/java/com/example/Banking/controller/AccountController.java

```diff
@@ -0,0 +1,43 @@
+package com.example.Banking.controller;
+import com.example.Banking.exception.IdNotFound;
+import com.example.Banking.model.Account;
+import com.example.Banking.service.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/Accounts")
+
```

Comment:
I prefer annotations to be just above class/method/variables without any space, its easier to read otherwise anyone can miss the annotation

Reply:
I removed the space and pushed the code